addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComLighting4Message.java

```diff
@@ -74,43 +80,56 @@ public static SubType fromByte(int input) throws RFXComUnsupportedValueException
     }
 
     public enum Commands {
-        UNDEFINED_0(0),
-        ON(1),
-        UNDEFINED_2(2),
-        UNDEFINED_3(3),
-        OFF(4);
+        OFF_0(0, false),
+        ON_1(1, true),
+        OFF_2(2, false),
+        ON_3(3, true),
+        OFF_4(4, false),
+        ON_5(5, true),
+        ON_7(7, true),
+        ON_9(9, true),
+        ON_12(12, true),
+        UNKNOWN(-1, false);
```

Comment:
If I store the command-ids of the on/off commands in the config then I can add all of them to lighting4.xml (and maybe a remark in the description to help us with filling in the gaps by reporting if you use the unknown ones..) The code can map the rest and send out the correct command.

And just saw that we only use 4 bytes for the command so that should 16 value, but still having 8 different unknown's does not look cool ;-)

However adding eight is not as bad as adding 50 ;-)